### DISCUSSION TITLE
[feat]: GitHub or Github!

### DISCUSSION THREAD

**Discussion by BaseMax** [pmndrs/react-spring#2045] [New Docs site feedback]
### A clear and concise description of what the feature is

Change Github to GitHub with an upper-case/capital H, not h.

https://www.react-spring.dev/

### Why should this feature be included?

GitHub is GitHub, not Github or github.

**Comment by joshuaellis** [comment]
I'm gonna move this to a discussion for someone else to pick up :)

**Reply by BaseMax** [reply]
Thanks, Josh @joshuaellis. If you share the path of the file that contains the components of the website, I can contribute and fix it with a PR.
